cookbooks/cdo-newrelic/attributes/default.rb

```diff
@@ -1,3 +1,6 @@
 default['cdo-newrelic'] = {
   'license-key'=>'0000000000000000000000000000000000000000',
+  'api-key'=>'00000000000000000000000000000000000000000000000',
```

Comment:
Could you add a comment explaining what the 0000000000000000000000 entries mean?